Dockerfile.rej

```
@@ -0,0 +1,85 @@
+diff a/Dockerfile b/Dockerfile	(rejected hunks)
```

Hadolint (2.14.0)

[error] unexpected 'd'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)